first-time-setup.md

```diff
@@ -0,0 +1,36 @@
+# First-Time Setup
```

Comment:
I believe the convention is to make the file all caps and snake case, so `FIRST_TIME_SETUP.md`